src/components/CreateRules/UploadRules.vue

```diff
@@ -19,81 +17,87 @@
-            class="help-section"
-            :key="index"
-          >
-            <span v-if="item.type == ItemTypes.TEXT" v-html="item.value"></span>
-            <span v-if="item.type == ItemTypes.PARTIAL_ITEMS">
-              <span v-for="(partialItem, index) in item.value" :key="index">
-                <span v-if="partialItem.type == ItemTypes.TEXT" v-html="partialItem.value"></span>
-                <a v-if="partialItem.type == ItemTypes.LINK"
-                   :href="partialItem.value.href"
-                   target="_blank"
-                >{{partialItem.value.linkText}}
-                  <v-icon dense color="primary">mdi-open-in-new</v-icon>
-                </a>
-              </span>
-            </span>
-          </li>
-        </ul>
-        <p id="help-text-section-3"
-          v-for="(item, index) in getCreateRulesResource.helpSection.helpText.section3.items"
-          class="help-section"
-          :key="index"
-        >{{ item }}
-        </p>
+          </p>
 
-        <u class="help-btn" @click="helpToggle = !helpToggle"><small>Hide Help</small></u>
-      </section>
-    </div>
+          <u class="help-btn" @click="helpToggle = !helpToggle"><small>Hide Help</small></u>
+        </section>
+      </div>
 
     <section id="confirm-rules-section" class="mt-10">
-      <header>
+      <header id="rules-confirm-header">
         <h2>2. Confirm Rules Completion</h2>
       </header>
-      <v-card flat id="confirm-rules-card">
-        <v-checkbox
-          class="chk-rules"
-          id="chk-confirm-rules"
-          :value="getCreateRulesStep.rulesConfirmed"
-          label="I confirm the following items are included as required in the Rules of the Association:"
-          @change="onRulesConfirmedChange($event)"
-        />
-        <ul>
-          <li>The Cooperative name is identified <b>exactly</b> as follows throughout the Memorandum:</li>
-          <p><b>{{getNameRequestDetails.approvedName}}</b></p>
-          <li>Each Subscriber and Witness has signed and dated the Rules of the Association and their name is
-            printed under their signature.</li>
-        </ul>
-      </v-card>
+      <div :class="{ 'invalid-section': getShowErrors && !hasRulesConfirmed }">
+        <v-card flat id="confirm-rules-card">
+          <v-form ref="confirmRulesChk">
+            <v-checkbox
+              class="chk-rules"
+              id="chk-confirm-rules"
+              :v-model="rulesConfirmed"
+              :value="getCreateRulesStep.rulesConfirmed"
+              :rules="confirmCompletionRules"
+              label="I confirm the following items are included as required in the Rules of the Association:"
+              @change="onRulesConfirmedChange($event)"
+            />
+            <ul>
+              <li>The Cooperative name is identified <b>exactly</b> as follows throughout the Memorandum:</li>
+              <div id="approved-name">{{getNameRequestDetails.approvedName}}</div>
```

Comment:
does another `<li>` element make sense here or is the div intended?

Reply:
or potentially a `<span>`?

Reply:
yeah it looks nasty like that.  this is what i'm trying to do...

![image](https://user-images.githubusercontent.com/6685223/132761362-6e6fb410-72eb-47fe-8c4a-25b2a7ee7095.png)

Reply:
ended up moving the `<div>` into the `<li>`.  also updated all the margin tweaks to use vuetify spacing helpers.

Reply:
LGTM